mathics/core/expression.py

```diff
@@ -2962,7 +2963,11 @@ def to_sympy(self, **kwargs):
         return None
 
     def to_python(self, *args, **kwargs) -> str:
-        return '"%s"' % self.value  # add quotes to distinguish from Symbols
+        if kwargs.get("string_quotes", True):
```

Comment:
Just in case, I shouldn't be useful to have `string_quotes` with `True` as default value, for backward compatibility?

Reply:
I don't understand. If you haven't set "string_quotes",  it defaults to True.  Almost all of the code currently is this way: no parameter is set.

If you pass this parameter, then you set it accordingly. 

I had thought about a better name, and at some in the past called it something else, but I can't remember the name.

Reply:
You are right! Discard my comment, please.